jest.config.mjs

```diff
@@ -0,0 +1,9 @@
+export default {
+  verbose: true,
+  testMatch: ['<rootDir>/test/**/*.?js'],
```

Comment:
**annotation**:
This is necessary because jest doesn't look for `.mjs` files by default.